eng/templates/azure-pipelines-shared-jobs.yml

```
@@ -0,0 +1,134 @@
+parameters:
+- name: isOfficial
+  displayName: Indicates whether to use official or unofficial 1ES templates
+  type: boolean
+  
+stages:
+  - stage: Build
+    dependsOn: []
+    displayName: Build
+    jobs:
+    - template: /eng/common/templates-official/jobs/jobs.yml@self
+      parameters:
+        artifacts:
+          publish:
+            logs: true
+            ${{ if in(variables['Build.SourceBranch'], 'refs/heads/main', 'refs/heads/production') }}:
```

This is essentially what `isOfficial` refers to, right? Might as well use it instead:
 ```suggestion
            ${{ if parameters.isOfficial }}:
```

I see below now that you do not treat them the same, but I believe they convey the same information of `this run will produce production-grade assets`